tests/api.c

```diff
@@ -34233,7 +34233,11 @@ static void test_wolfSSL_EVP_X_STATE_LEN(void)
     wolfSSL_EVP_CIPHER_CTX_init(ctx);
     AssertIntEQ(EVP_CipherInit(ctx, init, key, iv, 1), WOLFSSL_SUCCESS);
 
-    AssertIntEQ(wolfSSL_EVP_X_STATE_LEN(ctx), 272);
+    if (sizeof(void *) == 4)
```

Comment:
This seems problematic... This test is for ARC4 and expects it to be `sizeof(Arc4)` struct. Perhaps its better to just use that instead of this const value?

Reply:
Done.  Much better.